XenModel/XenSearch/Common.cs

```diff
@@ -1367,23 +1367,19 @@ public static Object GetImagesFor(PropertyNames p)
                             return Icons.XenCenter;
                         };
 
-                case PropertyNames.virtualisation_status:
+                case PropertyNames.virtualisation_status_for_search:

```

Comment:
This didn't need to change from a switch to an if. However, if you use virtualisation_status instead of virtualisation_status_for_search, it might be necessary to make the change.